### DISCUSSION TITLE
Does anyone know any python package that could communicate with Schlumberger Petrel ?

### DISCUSSION THREAD

**Discussion by yummy527** [GeoStat-Framework/GSTools#221] [Q&A] [answered]
Does anyone know any python package that could communicate with Schlumberger Petrel ?
Many geomodels in hand were build with Petrel...
Thanks.

**Comment by MuellerSeb** [comment, marked as answer]
Hey there,

Since Petrel is proprietary and only targets windows, we are not familiar with it. Maybe you can ask at https://softwareunderground.org

Cheers,
Sebastian

**Reply by yummy527** [reply]
Thanks!

**Comment by LSchueler** [comment]
Hi yummy527,

you are paying for a commercial and proprietary software, and I'm sure that also includes customer support. You should contact them about interfaces to open de facto standards like csv files (numpy is pretty good at reading them) or vtk data structures. Those are two examples of how you could use GSTools together with different tools.

**Reply by yummy527** [reply]
Thanks.
